src/data/defaultFeaturedItemData.ts

```diff
@@ -0,0 +1,40 @@
+const defaultFeaturedItem: any = {
```

Comment:
Here is a case where you can use the `FeaturedItemData` interface.

If you had a single object, you can type it as the following as an example:

```
const development: FeaturedItemData = {
  featuredItem: {
    imageID: "482815",
    backgroundImageSrc: "/482815.jpg",
    foregroundImageSrc: "/482815.jpg",
    uuid: "510d47d9-4f93-a3d9-e040-e00a18064a99",
    title: "Watuppa, From water front, Brooklyn",
    href: "https://qa-digitalcollections.nypl.org/items/510d47d9-4f93-a3d9-e040-e00a18064a99",
  },
  numberOfDigitizedItems: "875,861",
};
```

To be clear, DON'T change your data structure, it's fine but it needs an additional typescript utility type to help define the entire structure. What you can use is the `Record` utility type to help https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

I'll leave the rest for you to work through but let me know if you run into issues.